### ISSUE TITLE
Maintainers needed?

### ISSUE THREAD

**Issue by decentral1se** [YunoHost-Apps/mediawiki_ynh#6]
Hey, I'd like to be able to maintain this app.

I'll be relying on it quite a bit.

Up for giving me access to this repository?

**Comment by JimboJoe** [comment]
I've just invited you to the Mediawiki team :wink:

**Comment by anto1ne** [comment]
Pull Request https://github.com/YunoHost-Apps/mediawiki_ynh/pull/16 has been proposed last august and is still waiting to be merged 
I feel that a maintainer is needed for this app ...

**Comment by decentral1se** [comment]
Yes, probably does need it. I've merge #16 since it got another review.

**Comment by Salamandar** [comment]
@decentral1se I'd love to get it working on my machine (Rpi + Debian 10), that will also allow testing the Debian 10 support.
